app/dao/annual_billing_dao.py

```diff
@@ -49,3 +51,54 @@ def dao_get_all_free_sms_fragment_limit(service_id):
     return AnnualBilling.query.filter_by(
         service_id=service_id,
     ).order_by(AnnualBilling.financial_year_start).all()
+
+
+def set_default_free_allowance_for_service(service, year_start=None):
+    default_free_sms_fragment_limits = {
+        'central': {
```

Comment:
Worth looking at https://github.com/alphagov/notifications-admin/pull/3826#pullrequestreview-608658602 and https://github.com/alphagov/notifications-admin/pull/3830. The plan is that we don't have to maintain a list that we have to update every year.

Worth thinking what the implication of doing the same here would be for this PR

Reply:
I have a fix for that.